packages/mdc-select/mdc-select.scss

```diff
@@ -153,6 +155,16 @@
         @include mdc-theme-prop(color, text-primary-on-dark);
       }
     }
+
+    &--disabled {
```

Comment:
We no longer use this modifier class, nor do we set any style in `mdc-list` for a disabled list item. 👍